iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/AbstractTransformerSerDe.java

```diff
@@ -68,6 +69,7 @@ private static void writeTapChanger(TapChanger<?, ?, ?, ?> tc, NetworkSerializer
         context.getWriter().writeIntAttribute(ATTR_LOW_TAP_POSITION, tc.getLowTapPosition());
         var tp = tc.findTapPosition();
         context.getWriter().writeOptionalIntAttribute(ATTR_TAP_POSITION, tp.isPresent() ? tp.getAsInt() : null);
```

Comment:
For backward compatibility, if the **solved** tap position is present, it should be exported in the `"tapPosition"` field for IIDM version < 1.14.